qc-runner/src/classifier.r

```diff
@@ -69,7 +70,7 @@ task <- function(seurat_obj, config, task_name, sample_id){
 
   # Check wheter the filter is set to true or false
   # For some reason the last children of named lists are computed as vectors, so we can't access them as recursive objects. 
-  FDR <- as.numeric(config$filterSettings[["FDR"]])
+  FDR <- as.numeric(config$filterSettings[[tmp_sample]][["FDR"]])
```

Comment:
Is this always defined? Do we always send per-sample configuration?

Reply:
Actually the configuration list should be sample-specific to the current sample. Besides, tmp_sample is not defined at this point in the execution, we should look into this.

Reply:
`tmp_sample <- sub("sample-","",sample_id)` :-)

Reply:
Hmm I can't find the line

Reply:
Oh I hadn't collapsed the line